packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -424,4 +441,57 @@ export class GCFBootstrapper {
       });
     }
   }
+
+  /*
+   * Setting the process.env.WEBHOOK_TMP environment variable indicates
+   * that the webhook payload should be written to a tmp file in Cloud
+   * Storage. This allows us to circumvent the 100kb limit on Cloud Tasks.
+   *
+   * @param body
+   */
+  private async maybeWriteBodyToTmp(body: string): Promise<string> {
+    // Cloud tasks has a maximum payload size of 100kb, if a webhook payload is
+    // approaching this size, then serialize it to a temporary folder:
+    const MAX_PAYLOAD_SIZE = 65000;
```

Comment:
Can we do everything through the bucket? This adds an additional fork in the flow of data for the bots (tasks yes/no, storage yes/no). To keep it as simple as possible, should we always use a bucket?

Reply:
To be honest I had same thought reading this ... having two data flow paths felt complex to me